dev/Book-A-Meal/app/models/models.py

```diff
@@ -0,0 +1,62 @@
+class User(object):
+    def __init__(self, firstName=None, lastName=None, password = None):
+        self._firstName = firstName
+        self._lastName = lastName
+        self._password = password
+    
+    def firstName(self):
+        return self._firstName
+
+    def lastName(self):
+        return self._lastName
+    
+    def password(self):
```

Comment:
Don't think you need this method. Returning someone's password would ba security issue.